src/components/ResultsView.tsx

```diff
@@ -156,11 +162,13 @@ const ResultView = ({
   ctrlClick,
   views,
   extraColumn,
+  onRefresh,
 }: {
   r: Result;
   ctrlClick?: (e: Result) => void;
   views: { column: string; mode: string; value: string }[];
   extraColumn?: { row: (e: Result) => React.ReactNode; reserved: RegExp[] };
```

Comment:
similarly, we will one day get rid of `extraColumn` and `ctrlClick` in favor of the user being able to denote `actions` in columns.

Reply:
RE: `ctrlClick`

It might be a good idea to think about having a quicker way to achieve add to a canvas page or having `action:canvas` on by default on canvas pages.

I imagine if the only way a user could add to canvas via the drawer is by:

- clicking menu
- clicking column views
- clicking view
- clicking action
- typing `canvas`

This could get tedious each time.

Reply:
yea... maybe this should be a `Selection` instead of a View type? Would cut out all five of those steps.

The only weird thing would be how it interacts with views, but maybe we could filter it out of that process entirely

Reply:
Ok I like this alot more. Here's the selection
![Screen Shot 2023-05-05 at 2 58 07 PM](https://user-images.githubusercontent.com/7143571/236546026-7b3939ed-2d28-40f6-9c53-6d672286f033.png)

And here's the rendering. Notice that we could handle icons now!
![Screen Shot 2023-05-05 at 2 58 17 PM](https://user-images.githubusercontent.com/7143571/236546245-49c4bcb1-633f-4d08-9d9f-822e2f3b174b.png)

This gets a lot cooler with custom table width setting